common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -1022,46 +1022,6 @@ def test_check_button_checking_name(self):
         module = CapaFactory.create(attempts=1, max_attempts=10)
         self.assertEqual(module.check_button_checking_name(), "Checking...")
 
-        module = CapaFactory.create(attempts=10, max_attempts=10)
```

Comment:
Note to reviewers: I removed this assert because check_button_checking_name always returns the same thing (it does not depend on attempts or max_attempts).